src/components/QuranReader/SidebarNavigation/SidebarNavigation.module.scss

```diff
@@ -136,8 +140,14 @@
   }
 }
 
-.spaceOnTop {
-  padding-block-start: calc(3.5 * var(--spacing-mega));
+.spaceOnTopMenuExpanded {
```

Comment:
Opening the preview link in a new tab shows like this:

<img width="1724" alt="Screenshot 2024-06-25 at 3 00 26 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/0c67dcf3-8dad-41ef-8ee9-03848d9903d9">